sql/sql.y

```diff
@@ -278,6 +288,13 @@ func (e *expr) cdr() (r []string) {
 	return r
 }
 
+func (el exprlist) cdr() (r []string) {
```

Comment:
maybe need a meaningful name, what's `cdr` stands for?

Reply:
Change to `Strings()`